metadata-editor/app/controllers/EditsController.scala

```diff
@@ -158,7 +158,7 @@ object EditsController extends Controller with ArgoHelpers with DynamoEdits {
       .getOrElse(Future.failed(EditsValidationError("no-matching-metadata-found", "Couldn't find any matching metadata")))
     } recover {
       case NoItemFound => respondError(NotFound, "item-not-found", "Could not find image")
-      case e: EditsValidationError => respondError(BadRequest, e.key, e.message)
+      case e: EditsValidationError => respondError(NotFound, e.key, e.message)
```

Comment:
User: "Set metadata from usage rights?"
Computer: "No metadata for those usage rights found."